### PR TITLE
Facebook Auto Login

### PR DESCRIPTION
**Pull request by antrikshmisri**
# Description

This script allows user to automatically log into facebook with his/her credentails.

## Have you read the [Contributing Guidelines on Pull Requests](https://github.com/avinashkranjan/Amazing-Python-Scripts/blob/master/CONTRIBUTING.md)?

- [x] Yes
- [ ] No

## Type of change

_Please delete options that are not relevant._

- [x] New feature (non-breaking change which adds functionality)

## Checklist:

- [x] My code follows the style guidelines(Clean Code) of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have created a helpful and easy to understand `README.md`
- [x] My documentation follows [`Template for README.md`](https://github.com/avinashkranjan/Amazing-Python-Scripts/blob/master/Template%20for%20README.md)
- [x] My changes generate no new warnings
- [x] I have added tests/screenshots(if any) that prove my fix is effective or that my feature works.

### CODE REVIEW

File: Facebook-AutoLogin/README.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
+# Facebook-AutoLogin
+</br>
+
+## About This Project
+This is a python script that automates the facebook login process
+</br>
```
Comment (bismitaguha) [inline Facebook-AutoLogin/README.md:6]: Not needed

File: Facebook-AutoLogin/README.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
+# Facebook-AutoLogin
+</br>
```
Comment (bismitaguha) [inline Facebook-AutoLogin/README.md:2]: Not needed

**Review by bismitaguha — CHANGES_REQUESTED** [review]

File: Facebook-AutoLogin/README.md (hunk 1 above)
Comment (antrikshmisri) [reply Facebook-AutoLogin/README.md]: Just removed project name and about this project , will that do ?

File: Facebook-AutoLogin/README.md (hunk 2 above)
Comment (bismitaguha) [reply Facebook-AutoLogin/README.md]: I meant the <br> tag.

File: Facebook-AutoLogin/README.md (hunk 1 above)
Comment (bismitaguha) [reply Facebook-AutoLogin/README.md]: Remove the br tag

Comment (antrikshmisri) [reply Facebook-AutoLogin/README.md]: Oh ok  , removed the tag. Anything else?

**Comment by antrikshmisri** [comment]
@avinashkranjan  can you review the PR?

File: Facebook-AutoLogin/README.md (hunk 2 above)
Comment (antrikshmisri) [reply Facebook-AutoLogin/README.md]: @bismitaguha Made all the changes , can you re-review the PR

**Review by avinashkranjan — APPROVED** [review]

**Comment by avinashkranjan** [comment]
@all-contributors please add @antrikshmisri  for code and documentation

**Pull request merged**